packages/create-fuels/test/cli.test.ts

```diff
@@ -19,7 +19,7 @@ import { filterOriginalTemplateFiles, getAllFiles } from './utils/templateFiles'
 /**
  * @group node
  */
-describe('CLI', () => {
+describe('CLI', { timeout: 15_000 }, () => {
```

Comment:
Why did the timeout need to be increased here?

Reply:
I was getting timeouts consistently.